actionview/lib/action_view/helpers/form_tag_helper.rb

```diff
@@ -918,14 +918,16 @@ def extra_tags_for_form(html_options)
         end
 
         def form_tag_html(html_options)
+          tag = html_options["tag"] || "form"
```

Comment:
You can change this to `html_options.delete(“tag”)` to get the same behavior and prevent the tag attribute from appearing in the HTML.

Reply:
I changed it to `html_options.delete("wrapper_tag")` to better align with the helper you linked above.